x/exchange/keeper/invariants.go

```diff
@@ -75,6 +75,11 @@ func OrderStateInvariant(k Keeper) sdk.Invariant {
 				msg += fmt.Sprintf("\torder %d should have been deleted since it has no executable quantity\n", order.Id)
 				cnt++
 			}
+			if !order.RemainingDeposit.TruncateDec().Equal(order.RemainingDeposit) {
+				msg += fmt.Sprintf("\torder %d should have integer remaining deposit but has %s\n",
```

Comment:
Where should I look for logic to ensure that this part is always equal?

Reply:
That line checks that remaining deposit is an integer. Since `paid` in [here](https://github.com/crescent-network/crescent/pull/203/files#diff-ebffdcbfa19a1ef77c000e47489ed20e9300a4764320d9eeee8e589ccb87ee70R98) will always be an integer, [remaining deposit](https://github.com/crescent-network/crescent/pull/203/files#diff-ebffdcbfa19a1ef77c000e47489ed20e9300a4764320d9eeee8e589ccb87ee70R102) will always be an integer, too.